### PR TITLE
[docs] improvements related to cluster upgrades

### PR DESCRIPTION
**Pull request by nqb**
# Description
Improvements in docs related to cluster upgrades

# Impacts
Upgrade process

# Delete branch after merge
YES

### CODE REVIEW

**Comment by nqb** [comment]
Ready for review.

**Comment by nqb** [comment]
bump

File: UPGRADE.asciidoc [hunk 1]
```diff
@@ -72,6 +72,16 @@ You can take a backup of the pf directory with the following command:
...
 ----
 
+=== Disable monit alerts (only if you have monit installed)
+
+
+If `monit` is installed and running, shut it down with:
+
+[source,bash]
+----
+systemctl stop monit ; systemctl disable monit
```
Comment (julsemaan) [inline UPGRADE.asciidoc]: I'd put this on 2 lines to be consistent with the rest of the guide

**Comment by fdurand** [comment]
Can we have an include instead of a link.
I mean per example define the restart services section somewhere and include it in the section we need.
Most of the time when you follow a doc, you want to be able to copy and paste and follow step by step the procedure.

**Comment by nqb** [comment]
> Can we have an include instead of a link.

Yes, it's completely doable and certainly a better approach. But it requires to split `UPGRADE.asciidoc` in separate asciidoc files as @JeGoi did on #5936. 

At the moment, if you ok, I prefer to keep things as-is. Once #5936 is merged, I can split relevant sections in `UPGRADE.asciidoc` files and include them in `PacketFence_Cluster_Guide.asciidoc`.

**Comment by nqb** [comment]
I rebased on devel and move content to new directory structure defined by #5936.

**Pull request merged**